src/resources/config/shopify-app.php

```diff
@@ -326,6 +326,22 @@
 
     'billing_redirect' => env('SHOPIFY_BILLING_REDIRECT', '/billing/process'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Shopify Webhooks Address Type
+    |--------------------------------------------------------------------------
+    |
+    | This option is for defining webhooks type.
+    | `callbackUrl` it's use https:// route.
+    | `arn` it's use arn route.
+    |
+    | This documentation will helps you
+    | https://shopify.dev/apps/webhooks/configuration/eventbridge
+    |
+    */
+    
+    'webhook_address_type' => env('SHOPIFY_WEBHOOK_ADDRESS_TYPE', 'arn'),
```

Comment:
Can you please make `callbackUrl` the default. As a lot of current users will be using the standard webhook approach this allows for the easiest backwards compatibility.

Reply:
Hi @Kyon147 I have updated to `callbackUrl ` please check.

Thank you.